recipe/meta.yaml

```
@@ -33,11 +32,16 @@ requirements:
     - cloudpickle
     - fastprogress >=0.2.0
     - pandas >=0.24.0
-    - pytensor >=2.12.0,<2.13
-    - scipy >=1.4.1
+    - pytensor >=2.23,<2.24
+      # Reverting scipy to <1.13 due to pymc import failure that was caused due to older version of arviz
+    - scipy <1.13
```

```suggestion
    - scipy >=1.4.1,<1.13
```
https://github.com/pymc-devs/pymc/blob/v5.16.1/requirements.txt#L8
Although, what would it take to update arviz?

---

Doesn't seem to difficult looking at the recipe. Our highest version is `0.17.1` and current version I'm seeing on pypi is `0.18.0`